versions.props

```diff
@@ -17,8 +17,10 @@ com.github.ben-manes.caffeine:caffeine = 2.7.0
 org.apache.arrow:arrow-vector = 1.0.0
 org.apache.arrow:arrow-memory-netty = 1.0.0
 com.github.stephenc.findbugs:findbugs-annotations = 1.3.9-1
+com.amazonaws:aws-java-sdk-* = 1.11.749
```

Comment:
Should we version separately for S3 and future things that might be needed, like KMS etc?

Reply:
I believe most projects keep the sdk package dependencies in sync to avoid any issues with some of the core/api packages evolving.  I feel like we should keep them in sync for now and diverge if the need arises in the future.